src/modules/contact/components/form/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from "react";
+import styles from "./style.module.css";
+
+const initialState = {
+  firstName: "",
+  lastName: "",
+  phone: "",
+  email: "",
+  subject: "",
+  message: ""
+}
+
+class Form extends Component {
+  constructor() {
+    super();
+    this.state = initialState
+  }
+
+  onFormSubmit = (event) => {
```

Comment:
1. The name of the method is not appropriate, you can make it `sendMessage` to be more descriptive.

2. You should use `axios` not `fetch`.

3. The request function should be in the `services` not in the component.

4. Show success/error message  (feedback) to the user not in the console.